Can't create a new project using create-directus-project

### Describe the Bug

I am trying to create a new project using 
```
npx create-directus-project example-project
```

But I am getting the following error:

```bash
npx: installed 57 in 4.621s
internal/modules/cjs/loader.js:888
  throw err;
  ^

Error: Cannot find module 'object-hash'
- D:\coding\learning\directus\mypr2\node_modules\directus\dist\services\assets.js
- D:\coding\learning\directus\mypr2\node_modules\directus\dist\services\index.js
- D:\coding\learning\directus\mypr2\node_modules\directus\dist\utils\get-schema.js
- D:\coding\learning\directus\mypr2\node_modules\directus\dist\cli\commands\bootstrap\index.js
- D:\coding\learning\directus\mypr2\node_modules\directus\dist\cli\index.js
- D:\coding\learning\directus\mypr2\node_modules\directus\cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\coding\learning\directus\mypr2\node_modules\directus\dist\services\assets.js:28:39)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\coding\\learning\\directus\\mypr2\\node_modules\\directus\\dist\\services\\assets.js',
    'D:\\coding\\learning\\directus\\mypr2\\node_modules\\directus\\dist\\services\\index.js',
    'D:\\coding\\learning\\directus\\mypr2\\node_modules\\directus\\dist\\utils\\get-schema.js',
    'D:\\coding\\learning\\directus\\mypr2\\node_modules\\directus\\dist\\cli\\commands\\bootstrap\\index.js',
    'D:\\coding\\learning\\directus\\mypr2\\node_modules\\directus\\dist\\cli\\index.js',
    'D:\\coding\\learning\\directus\\mypr2\\node_modules\\directus\\cli.js'
  ]
}
✖ Installing Directus
Error: undefined
```

### To Reproduce

Following the [quickstart guide](https://docs.directus.io/getting-started/quickstart/) and getting the error in the first step

npx create-directus-project example-project


### What version of Directus are you using?

9.0.0-rc.86

### What version of Node.js are you using?

14.17.0 and 14.17.3

### What database are you using?

none, throw error before the DB configuration process

### What browser are you using?

Edge

### What operating system are you using?

Windows 10

### How are you deploying Directus?

Running locally

--- a/api/package.json
+++ b/api/package.json
@@ -124,6 +124,7 @@
 		"node-cron": "^3.0.0",
 		"node-machine-id": "^1.1.12",
 		"nodemailer": "^6.6.1",
+		"object-hash": "^2.2.0",
 		"openapi3-ts": "^2.0.0",
 		"ora": "^5.4.0",
 		"otplib": "^12.0.1",
@@ -151,7 +152,6 @@
 		"memcached": "^2.2.2",
 		"mysql": "^2.18.1",
 		"nodemailer-mailgun-transport": "^2.1.3",
-		"object-hash": "^2.2.0",
 		"pg": "^8.6.0",
 		"sqlite3": "^5.0.2",
 		"tedious": "^11.0.8"

